### PR TITLE
GHA: CI bring in arm64 k8s CI

### PR DESCRIPTION
**Pull request by jongwu**
This PR is just a rebase of @fidencio's [PR](https://github.com/kata-containers/kata-containers/pull/8236)

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -36,6 +36,27 @@ jobs:
...
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Push multi-arch manifest
+        run: |
+          docker manifest create ghcr.io/kata-containers/kata-deploy-ci:${{ inputs.tag }} \
+          --amend ghcr.io/kata-containers/kata-deploy-ci:${{ inputs.tag }}-amd64 \
+          --amend ghcr.io/kata-containers/kata-deploy-ci:${{ inputs.tag }}-arm64
+          docker manifest push quay.io/kata-containers/kata-deploy-ci:${{ inputs.tag }}
```
Comment (stevenhorsman) [inline .github/workflows/ci.yaml:58]: Is this a reason that we aren't building teh s390x and ppc64le images into the manifest, like we do in payload-after-push: https://github.com/kata-containers/kata-containers/blob/b056683b7a4e6a57e3526955ff9c7f4cca779450/.github/workflows/payload-after-push.yaml#L104-L111 ?

**Review by GabyCT — APPROVED** [review]

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks

**Comment by jongwu** [comment]
Hi @fidencio @bergwolf  -,

I'm sorry to tell you that  for some reason there will be no one continue to maintain the current Arm CI nodes. So, It's not safe to deploy test there and this will impact all CI on Arm. I hope there will be volunteers can take this. Sorry about that.

**Pull request closed without merging**